delfin/drivers/dell_emc/vmax/client.py

```diff
@@ -28,9 +28,16 @@
 class VMAXClient(object):
     """ Client class for communicating with VMAX storage """
 
-    def __init__(self):
+    def __init__(self, **kwargs):
         self.conn = None
         self.array_id = None
+        rest_access = kwargs.get('rest_access')
+        if rest_access is None:
+            raise exception.InvalidInput('Input rest_access is missing')
+        self.rest_host = rest_access.get('host', 'localhost')
+        self.rest_port = rest_access.get('port', '8088')
```

Comment:
Why is this hardcoded?
 If it always be 8088, then we can keep it in CONSTANT

Reply:
As host and port are required, and it was checked when creating access info, so I just removed those default value.